app/models/competition_tab.rb

```diff
@@ -34,6 +34,16 @@ def slug
     competition.tabs.where("display_order > ?", display_order).update_all("display_order = display_order - 1")
   end
 
+  validate :verify_if_urls_are_full_urls, on: :update
+  private def verify_if_urls_are_full_urls
```

Comment:
`verify_full_urls` should do, maybe I can be convinced to call it `verify_if_full_urls`. But you don't have to turn the method name into a whole English sentence :wink:

Reply:
Okay used `verify_if_full_urls` then :-)